rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/utils/misc.kt

```diff
@@ -70,6 +70,16 @@ fun String?.asDouble(): Double {
         return 0.0
     }
 }
+fun String?.asBigDecimal(): BigDecimal? {
```

Comment:
I would put a blank line before this one.
Also, why supporting `String?` instead of `String` if anyway we want to return null when the string is null?
One could support just string and then use it as `myStringValuePossiblyNull?.asBigDecimal()`. This would result in a bit less code in this function

Reply:
Oh, yes, good idea!